.circleci/config.yml

```diff
@@ -25,30 +25,33 @@ runOnDocker: &runOnDocker
   working_directory: /go/src/github.com/mesg-foundation/core
 
 jobs:
-  "build_docker":
-    <<: *runOnMachine
+  "unit_test":
+    <<: *runOnDocker
     steps:
       - checkout
-      - run: docker pull mesg/core:latest
-      - run: docker build -t mesg/core:$CIRCLE_SHA1 --build-arg version="dev build `echo $CIRCLE_SHA1 | cut -c1-7`" .
-
-  "test":
+      - setup_remote_docker # this should be remove when no test are actually using docker
+      - run: go get -t ./...
```

Comment:
Remove go get

Reply:
yes good point like that we will directly see if we forgot to put some stuff on the vendor folder